simplified-cardcreator/src/test/kotlin/org/nypl/simplified/cardcreator/ui/ConfirmationFragmentTest.kt

```diff
@@ -90,6 +93,31 @@ class ConfirmationFragmentTest {
     )
   }
 
+  @After
+  fun tearDown() {
+    /*
+    * There are some issues with activityscenario state when an Activity's finish() is called
+    * at certain lifecycle points so we only call close on non-finishing activities to avoid this.
+    * Not closing the scenarios invoking finish does introduce potential waste, but as of right now
+    * there is only this one test involving a finish() call.
+    *
+    * Reportedly this issue might be fixed by upgrading AndroidX Test Core to 1.5+
```

Comment:
Would upgrading upgrading AndroidX Test Core to 1.5+ be alot of overhead?

Reply:
I don't think the `relaxed=true` or `scenario.close()` saved us much time if any. They're moreso housekeeping additions. Ultimately I wasn't able to figure out any obvious way to make things measurably quicker.

The `relaxed=true` was to address some random exceptions that were being thrown for certain tests where methods that weren't stubbed were being called indirectly by the test code. 

The `scenario.close()` is actually a change that could result in a performance increase, but we just dont have that many scenario tests yet. I think you'd see a performance improvement with a large suite of tests (like hundreds).  For us the improvement is likely very minimal and this more just to have proper tests that clean up after themselves.

As for AndroidX Test Core 1.5+ I'm not so sure, I haven't looked into if there's any significant or breaking changes but I imagine it would be an easy upgrade. Currently however the 1.5+ builds are in an early alpha stage, so I figured we could wait a bit and just use this janky fix for the time being given it's only for a single test.